src/steps.tsx

```diff
@@ -369,8 +386,10 @@ const riskAssessment: Step[] = [
   },
   {
     id: 'askHaveDialogue',
-    message:
-      'Ok great. Thank you for your answers. Last question, do you have access to Dialogue through your employer?',
+    message: `
+      Ok great. Thank you for your answers. 
+
+      Last question, do you have access to Dialogue through your employer?`,
```

Comment:
could people not know what Dialogue is?

Reply:
jk, didn't see question.

I still feel like it might come off as gratuitous promotion in the B2C context though (talking about Dialogue in the chat)

Reply:
Yeah, that's why I originally didn't add it. @alexissmirnov would you be okay to remove it?

Reply:
Flow is already pretty long